src/core/hle/service/set/set_sys.h

```diff
@@ -269,6 +269,16 @@ class SET_SYS final : public ServiceFramework<SET_SYS> {
     };
     static_assert(sizeof(EulaVersion) == 0x30, "EulaVersion is incorrect size");
 
+    /// This is nn::settings::system::HomeMenuScheme
+    struct HomeMenuScheme {
+        u32 Main;
```

Comment:
```suggestion
        u32 main;
```
Same for the other struct members